jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/conf/JdbcStorageConfigManager.java

```diff
@@ -28,28 +37,38 @@
  */
 public class JdbcStorageConfigManager {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(JdbcStorageConfigManager.class);
   public static final String CONFIG_PREFIX = "hive.sql";
+  public static final String CONFIG_PWD = CONFIG_PREFIX + ".dbcp.password";
   private static final EnumSet<JdbcStorageConfig> DEFAULT_REQUIRED_PROPERTIES =
     EnumSet.of(JdbcStorageConfig.DATABASE_TYPE,
```

Comment:
i mean, non metastore case requires additional parameters. We were checking for additional params and giving appropriate error in earlier case, but not anymore. (Please let me know if some other check would give appropriate error).